src/fstab-generator/fstab-generator.c

```diff
@@ -110,15 +110,15 @@ static int mount_array_add_internal(
                 char *in_what,
                 char *in_where,
                 const char *in_fstype,
-                const char *in_options) {
+                char *in_options) {
```

Comment:
Then please use `TAKE_PTR()` in the caller for the option.
Otherwise, it will be freed after assigned.

Reply:
I just figured that out and force-pushed :-) Thanks for the tip! I'm a still a newbie in the systemd codebase sorry!